clusters/tool.json

```diff
@@ -9138,7 +9157,7 @@
       },
       "related": [],
       "uuid": "be419332-61ab-45f0-979e-56f78a223c8e",
-      "value": "metasploit"
+      "value": "METASPLOIT is a penetration testing framework whose features include vulnerability testing, network enumeration, payload generation and execution, and defense evasion. Availability: Public"
```

Comment:
Shouldn't this be in the "description" field instead?

Reply:
Woops indeed I'll fix that. thanks for catching it!